packages/service-library/src/servicelib/exception_utils.py

```diff
@@ -58,7 +57,7 @@ def try_to_raise(self, exception: BaseException) -> None:
             raise exception
 
         # ignore if exception inside delay window
-        log.warning("%s skip(s) of exception: %s", self._failure_counter, exception)
+        log.info("%s skip(s) of exception: %s", self._failure_counter, exception)
```

Comment:
question. why do we have a log here at all? the exception is raised. so the client should do whatever it wants with this no? any input on this @pcrespov ?

Reply:
Since I have also posed myself this question and cannot answer to it any longer. I will just drop this message, since it provides zero insight into what is happening.